### PR TITLE
Sentry

### PR DESCRIPTION
**Pull request by kevinaltschuler**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,8 +2,24 @@
...
+import { Integrations } from '@sentry/tracing';
+
 import App from './components/App';
 
 require('./favicon.ico'); // Tell webpack to load favicon.ico
 
+if (process.env.NODE_ENV === 'production' || true) {
+  Sentry.init({
+    dsn:
+      'https://ad5d766ba9fb46d784ee1418d837605b@o561173.ingest.sentry.io/5697727',
```
Comment (kevinaltschuler) [inline src/index.js:15]: this key should be fine to be public on github

Comment (kevinaltschuler) [reply src/index.js:15]: bad actors could send you fake errors but thats all

Comment (atc3) [reply src/index.js:15]: ok as long as we have the option of regenerating the secret

**Review by atc3 — APPROVED** [review]
LGTM

**Pull request merged**